language-support/scala/codegen-sample-app/src/test/scala/com/digitalasset/codegen/ScalaCodeGenIT.scala

```diff
@@ -284,6 +284,7 @@ class ScalaCodeGenIT
   }
 
   "alice creates TemplateWith23Arguments contract and receives corresponding event" in {
+    //noinspection NameBooleanParameters
```

Comment:
I believe you can disable this in your IDE without adding comments.

Reply:
Not selectively; it's all or nothing, AFAIK. If it's annoying to others, I can remove it again. What do you think?